draft-ietf-dnsop-generalized-notify.md

```diff
@@ -470,29 +461,64 @@ Alternative solutions (such as signing notifications and validating
 their signatures) appear significantly more expensive without tangible
 benefit.
 
+The specification does not require DSYNC records to be signed with
```

Comment:
It obviously doesn't make any sense to NOT sign a particular record (like the DSYNC RRset), so saying that this is not required is sort of beside the point.

I would rephrase this as "...does not require the parent zone to be DNSSEC signed."

Reply:
Yeah. I tried to avoid "parent zone" because in this document this term is usually used for the zone's parent zone, not for the enclosing zone. But I agree with you, and will come up with something.

Reply:
done